src/Mod/PartDesign/Gui/TaskPrimitiveParameters.cpp

```diff
@@ -630,75 +633,77 @@ void TaskBoxPrimitives::onWedgeXminChanged(double v) {
-    ui->wedgeXmin->setMaximum(v); // must be <= than wedgeXmax
+    ui->wedgeXmin->setMaximum(v); // must be < than wedgeXmax
     sph->Xmax.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeYminChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeYmax->setMinimum(v);
+    ui->wedgeYmax->setMinimum(v); // must be > than wedgeYmin
     sph->Ymin.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeYmaxChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeYmin->setMaximum(v);
+    ui->wedgeYmin->setMaximum(v); // must be < than wedgeYmax
     sph->Ymax.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeZ2minChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeZ2max->setMinimum(v);
+    ui->wedgeZ2max->setMinimum(v); // must be >= than wedgeZ2min
     sph->Z2min.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeZ2maxChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeZ2min->setMaximum(v); // must be <= than wedgeXmax
+    ui->wedgeZ2min->setMaximum(v); // must be <= than wedgeZ2max
     sph->Z2max.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeZminChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeZmax->setMinimum(v);
+    ui->wedgeZmax->setMinimum(v); // must be > than wedgeZmin
     sph->Zmin.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 void TaskBoxPrimitives::onWedgeZmaxChanged(double v) {
     PartDesign::Wedge* sph = static_cast<PartDesign::Wedge*>(vp->getObject());
-    ui->wedgeZmin->setMaximum(v);
+    ui->wedgeZmin->setMaximum(v); // must be < than wedgeZmax
     sph->Zmax.setValue(v);
     vp->getObject()->getDocument()->recomputeFeature(vp->getObject());
 }
 
 
-
-
-void  TaskBoxPrimitives::setPrimitive(App::DocumentObject *obj)
+bool TaskBoxPrimitives::setPrimitive(App::DocumentObject *obj)
 {
     try {
         QString name(QString::fromLatin1(Gui::Command::getObjectCmd(obj).c_str()));
         QString cmd;
         App::Document* doc = App::GetApplication().getActiveDocument();
         if (!doc) {
-            return;
+            return false;
         }
+        // To allow e.g. a width of 2.2 um while the users has e.g. 2 user decimals
+        // we cannot use Base::UnitsApi::getDecimals().
+        // It is also more sensible to to disallow values OCC allows, thus use OCC's limit

```

Comment:
There is a typo here: a repeated "to". I also don't follow the stated logic: do you mean "disallow values OCC **dis**allows"?
```suggestion
        // It is also more sensible to disallow values OCC (dis?)allows, thus use OCC's limit
```

Reply:
Many thanks for the review.

I fixed now the typos and wording as you suggested.